order-service/src/main/java/com/innowise/orderservice/config/KafkaConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.innowise.orderservice.config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+
+@Configuration
+public class KafkaConfig {
```

Comment:
Missing payments topic creation in OrderService KafkaConfig:
Only creates orders topic (line 30)
Should also ensure payments topic exists for consumer
Missing producer/consumer error handlers configuration